src/esbmc/options.cpp

```diff
@@ -97,7 +97,8 @@ const struct group_opt_templ all_cmd_options[] = {
     {"warning,W",
      boost::program_options::value<std::vector<std::string>>(),
      "enable specific frontend warnings, disable with \"no-\" prefix, or pass "
-     "options directly to the C/C++ frontends with the form -Wc,OPT1,OPT2,..."},
+     "options directly to the C/C++ frontends with the form "
+     "-Wc,OPT1,OPT2,..."},
```

Comment:
is this change needed?

Reply:
I do not know why Rafael made this change but I have reverted it.